### PR TITLE
[MILESTONE 6] View

### PR DESCRIPTION
**Pull request by Maxwell011**
Hello in this PR I did the following 👍 

-  Created Styling for view (It was a minimalistic design)
- Used methods that I created in the Blog App

Kindly Review Thank you 🙏

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,27 @@
...
+      <p><%= post.text %></p>
+      <h4>Comments <%= post.comments_counter %> likes: <%= post.likes_counter %></h4>
+    </div>
+    <% @comments=Comment.where(post_id: post.id)%>
+    <div id="comments_container">
+    <% @comments.each do |comment| %>
+        <div id="comments">
+         <%= @user.name%>: <%= comment.text%>
+        </div>
+    <% end %>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb]: - [ ] Kindly use the `last_five_comments` callback function to render the most recent comments for a post.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,24 @@
...
+  <% @user.last_three_posts.each do |post|%>
+    <div id="users_id">
+      <h3>Post #<%= post.id %></h3>
+      <p><%= post.text %></p>
+      <h4>Comments <%= post.comments_counter %> likes: <%= post.likes_counter %></h4>
+    </div>
+  <% end %>
+  <div id="button_container">
+    <button id="see_all_button"><%= link_to "See all posts", "/" %></button>
+  </div>
```
Comment (elisha2kyakpo1) [inline app/views/users/show.html.erb]: - [ ] This button should take users to the post index page and not the home page.

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+    @user = User.find(params[:id])
+    @posts = Post.where(user_id: @user.id)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = Post.where(user_id: @user.id)
+    @post = Post.find(params[:post_id])
+  end
```
Comment (elisha2kyakpo1) [inline app/controllers/posts_controller.rb]: - [ ] Kindly fix the show action for post(s) post. I have posts I created on the Rails console, when I try to fetch a specific post using `http://localhost:3000/users/1/posts/1` I got an error.

image

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1 +1,27 @@
...
+<div id="posts_container">
+  <div id="user_container">
+  <img id="user_img" src="<%= @user.photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= @user.name %></h3>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+    <% @posts[0...5].each do |post|%>
+    <div id="user_posts">
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb]: - [ ] The range loop prevents the display of all the posts using the button `See all posts`.

all-posts

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Maxwell011, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (Maxwell011) [reply app/views/posts/index.html.erb]: Done

**Review by sumairq — APPROVED** [review]
### Hi @Maxwell011 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**